src/workloads/execution/MultiPlanning.yml

```diff
@@ -0,0 +1,148 @@
+- &Key {^FastRandomString: {length: 6, alphabet: "0123456789"}}
+- &SmallKey {^Join: {array: ["1", {^FastRandomString: {length: 5, alphabet: "0123456789"}}]}}
+- &BigKey {^Join: {array: ["8", {^FastRandomString: {length: 5, alphabet: "0123456789"}}]}}
+- &SelectiveQuery {^Choose: {
+  from: [
+    {$gte: *BigKey},
+    {$lte: *SmallKey}
+  ]
+}}
+- &NonSelectiveQuery {^Choose: {
+  from: [
+    {$gte: *SmallKey},
+    {$lte: *BigKey}
+  ]
+}}
+
+Actors:
+- Name: InitCollectionAndDropPlanCache
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        drop: *Collection
+    - OperationName: RunCommand
+      OperationCommand:
+        createIndexes: *Collection
+        indexes:
+        - key:
+            a: 1
+          name: a_1
+        - key:
+            b: 1
+          name: b_1
+        - key:
+            c: 1
+          name: c_1
+  - &Nop {Nop: true}
+  - Blocking: None
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        planCacheClear: *Collection
+  - Blocking: None
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        planCacheClear: *Collection
+  - Blocking: None
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        planCacheClear: *Collection
+
+- Name: Insert
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Threads: 1
+        CollectionCount: 1
+        DocumentCount: *docCount
+        BatchSize: 1000
+        Document:
+          a: *Key
+          b: *Key
+          c: *Key
+
+- Name: Lookup
+  Type: CrudActor
+  Database: *Database
+  Threads: 4
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 2000
+    Collection: *Collection
+    Operations:
+    - OperationName: find
+      OperationMetricsName: FindWithMostSelectiveA
+      OperationCommand:
+        Filter: {
+          a: *SelectiveQuery,
+          b: *NonSelectiveQuery,
+          c: *NonSelectiveQuery
+        }
+  - Repeat: 2000
+    Collection: *Collection
+    Operations:
+    - OperationName: find
+      OperationMetricsName: FindWithMostSelectiveB
+      OperationCommand:
+        Filter: {
+          a: *NonSelectiveQuery,
+          b: *SelectiveQuery,
+          c: *NonSelectiveQuery
+        }
+  - Repeat: 2000
+    Collection: *Collection
+    Operations:
+    - OperationName: find
+      OperationMetricsName: FindWithMostSelectiveC
+      OperationCommand:
+        Filter: {
+          a: *NonSelectiveQuery,
+          b: *SelectiveQuery,
+          c: *NonSelectiveQuery
+        }
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone
+      - standalone-all-feature-flags
+      - replica
+      - replica-all-feature-flags
```

Comment:
I think this list is wrong. It seems sufficient to run this test just against our standalone variants, not against our 1-node replica set variants. However, we definitely want this to run against the classic and SBE specific build variants. I believe you can use the same list as the recently added $group spill-to-disk tests:

https://github.com/mongodb/genny/blob/9ee62c040a2bc7609d566040f012140daf8d81b1/src/workloads/query/GroupSpillToDisk.yml#L132-L135

Reply:
Done